Assets/Scripts/Copilots/HealingBubble.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealingBubble : MonoBehaviour
+{
+    //variables for spore particle system
+    public float sporeTimer;
+    private bool sporeStart;
+    private ParticleSystem spores;
+
+    //variables for healing bubble
+    private MeshRenderer bubbleMesh;
+    private Vector3 bubbleScale;
+    private bool bubbleInflated;
+
+    //variables for healing player
+    private bool healPlayer;
+    private float healTime;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        transform.localPosition = Vector3.zero;
+        sporeStart = false;
+        spores = GetComponent<ParticleSystem>();
+        spores.Stop();
+        bubbleMesh = transform.GetComponent<MeshRenderer>();
+        bubbleInflated = false;
+        bubbleScale = new Vector3(0.01f, 0.01f, 0.01f);
```

Comment:
I noticed this was another hard coded value and was wondering if you could change bubbleScale to be public and set the value there rather than here